### DISCUSSION TITLE
New features  && suggestions

### DISCUSSION THREAD

**Discussion by 0tt049** [0tt049/dev4dev#74] [Ideas]
- Use github api to populate seed
  - Get avatars
  - Get main programming language
- Refactor stylesheet following rails convention
  - All changes to  .scss files must go to `app/assets/stylesheets/config/*.scss`
  - Go all in, so we can reuse it on Murdoc 
- Implement search bar
  - Filter by language
  - Filter by anything
- Set slots default to "available" so that gigs, when booked, will block-out that date-range out of slots.
  - Make default to opt-out instead of opt-in. Like Airbnb
  - Implement realtime notification ([Action-Cable](https://guides.rubyonrails.org/action_cable_overview.html))
- Plan presentation
- Add language logo
- Block-out booked slots from index 

Please, add your suggestions and let's vote!
@vcanedo @praecipito @Gmmuoio

**Comment by praecipito** [comment]
Priorities suggestions:
- Edit/update and destroy actions to gigs.
- Fix date/time validations.

**Comment by 0tt049** [comment]
### Fixes

- Complete Gigs CRUD (edit/update, destroy) 
- Implement Date/Time validations

### Features
#### Must have

- Search
  - Filter by language
  - Opt. Filter by anything
  - Block-out booked slots from index

- Seed
  - Use github API to populate seed
  - Make sure seed will pass validations
  - Get user avatar
  - Opt. get user most used language

- UI
  - Add language logo
  - All changes to .scss files must go to app/assets/stylesheets/config/*.scss
  - Go all in, so we can reuse it on Murdoc

- Model 
  - Set slots default to "available" so that gigs, when booked, will block-out that date-range out of slots.
  - Make default to opt-out instead of opt-in. Like Airbnb
  - Implement realtime notification ([Action-Cable](https://guides.rubyonrails.org/action_cable_overview.html))

- Plan presentation
